TaskHouseApi/Model/CategorySkill.cs

```diff
@@ -0,0 +1,11 @@
+namespace TaskHouseApi.Model
+{
+    public class CategorySkill
```

Comment:
Is this the way to describe many to many relationships?

Reply:
Yes:
https://www.learnentityframeworkcore.com/configuration/many-to-many-relationship-configuration

Reply:
Super